application/comit_node/src/lib.rs

```diff
@@ -8,9 +8,27 @@ pub mod comit_client;
 pub mod comit_i_routes;
 pub mod http_api;
 pub mod libp2p_bam;
+pub mod load_settings;
 pub mod logging;
 pub mod network;
 pub mod node_id;
 pub mod seed;
 pub mod settings;
 pub mod swap_protocols;
+
+fn var_or_default(name: &str, default: String) -> String {
+    match std::env::var(name) {
+        Ok(value) => {
+            log::info!("Set {}={}", name, value);
```

Comment:
Shall we set a different log level for this, e.g. `debug` or `trace`?

Reply:
I'd keep it like this for now as it says where comit_node get the config from.
It'll then get trashed with #947.

Reply:
True. I keep on forgetting about the other ticket...